contrib/guix/manifest.scm

```diff
@@ -485,12 +485,9 @@ inspecting signatures in Mach-O binaries.")
           `(append ,flags
             ;; https://gcc.gnu.org/install/configure.html
             (list "--enable-threads=posix",
-                  building-on)))
-        ((#:make-flags flags)
-          ;; Uses the SSP functions from glibc instead of from libssp.so.
-          ;; Our 'symbol-check' script will complain if we link against libssp.so,
-          ;; and thus will ensure that this works properly.
-          `(cons "gcc_cv_libc_provides_ssp=yes" ,flags))))))
+                  "--enable-default-ssp=yes",
+                  "--enable-default-pie=yes",
```

Comment:
How does this affect binaries? Aren't position independent executables an ELF thing, not PE?

Reply:
Pretty sure I did this so we could consolidate the gcc hardening options, doesn't need to be a blocker for this, so droppe for now.